sdk/cosmos/azure-cosmos-spark_3-1_2-12/README.md

```diff
@@ -27,6 +27,8 @@ https://github.com/Azure/azure-sdk-for-java/issues/new
 #### azure-cosmos-spark_3-1_2-12
```

Comment:
curious, why we need to mention both version? 3-1 and 3-2? since there is one readme.md for each version

Reply:
T avoid customers going to the same location they used before (Spark 3.1 readme/changelog) to get confused on how to move to Spark 3.2 - at least that was the idea behind it